src/app/modules/common/layouts/admin-layout/admin-layout.component.scss

```diff
@@ -58,9 +59,17 @@
   top: 13px;
 }
 
+.logo:focus-visible,
+.logo-text:focus-visible {
+  background: rgba(120, 120, 120, 0.2);
+  box-shadow: 0 0 0 1px rgb(255, 255, 255, 0.3) inset;
```

Comment:
We should store these values in variables so they're are easily changeable. If I want to change the focus color of a theme, I should just have to change the value of that css variable. See how `primary` `accent` and other colors are declared project wide for different themes.